src/javascript/ContentEditor.register.jsx

```diff
@@ -5,6 +5,7 @@ import {Constants} from '~/ContentEditor.constants';
 import ContentEditorApi from '~/Api/ContentEditor.api';
 import ContentEditorRedux from './ContentEditor.redux';
 import {ContentEditorHistoryContextProvider} from '~/ContentEditorHistory/ContentEditorHistory.context';
+import {removeMixinFromSection, addMixinToSection} from '~/ContentEditor.register.Utils';
```

Comment:
As we want customers to use the utility function, can we pass the utility functions as an onChange parameter?